lib/scalingo/api/response.rb

```diff
@@ -1,7 +1,24 @@
 module Scalingo
   module API
     class Response
-      def self.unpack(client, response, key: nil)
+      def self.timeout(client)
+        new(
+          client: client,
+          status: nil,
```

Comment:
Should we add another status for timeout ? There might be other reason for an empty status (DNS resolution failed, connection reset, etc...)